sermant-plugins/sermant-tag-transmission/tag-transmission-dubbo2.7.x-plugin/src/main/java/com/huaweicloud/sermant/tag/transmission/apachedubbov2/interceptors/ApacheDubboConsumerInterceptor.java

```diff
@@ -40,7 +40,7 @@ public class ApacheDubboConsumerInterceptor extends AbstractClientInterceptor<Rp
     private static final Logger LOGGER = LoggerFactory.getLogger();
 
     /**
-     * rpcInvocation参数在invoke方法的参数下标
+     * rpcInvocation parameter subscript
```

Comment:
rpcInvocation parameter subscript of the invoke method

Reply:
done